### ISSUE TITLE
Stuck after choosing document country

### ISSUE THREAD

**Issue by domasn** [Veriff/veriff-sample-ios#1]
Veriff version `2.0.2`

After choosing document country the SDK gets stuck with a spinner:
<img src="https://user-images.githubusercontent.com/7569279/53965590-2ba8df80-40fa-11e9-86ab-8c60c4b1ce5f.png" width=300>

At the same time I can see things printed in the debug console:
```
POST: https://magic.veriff.me/v1/verifications/{XXX}/event resp Optional("{}")
JSON parsing error: Error Domain=NSCocoaErrorDomain Code=4864 "Expected to decode String but found a dictionary instead." UserInfo={NSCodingPath=(
), NSDebugDescription=Expected to decode String but found a dictionary instead.}
GET: https://magic.veriff.me/v1/mobile/{XXX} resp Optional("{\"status\":\"success\",\"verification\":{\"id\":\"dae2fb6a-XXXX-XXXX-XXXX-946a33ab4ebb\",\"state\":\"selfid\",\"callback\":\"\",\"tosUrl\":\"https://veriff.me/tos\",\"tosName\":null,\"flowLayout\":\"\",\"vendorCapabilities\":[\"ID_CARD\",\"PASSPORT\",\"RESIDENCE_PERMIT\",\"DRIVERS_LICENSE\"],\"isInterview\":false,\"documentType\":\"ID_CARD\",\"supportedDocumentTypes\":[\"ID_CARD\",\"PASSPORT\",\"RESIDENCE_PERMIT\",\"DRIVERS_LICENSE\"]}}")
JSON parsing error: Error Domain=NSCocoaErrorDomain Code=4865 "No value associated with key CodingKeys(stringValue: "lang", intValue: nil) ("lang")." UserInfo={NSCodingPath=(
), NSDebugDescription=No value associated with key CodingKeys(stringValue: "lang", intValue: nil) ("lang").}
```

Seems like models are out sync and response can't be parsed.

Thank you @kristjanpirnpuu 
Waiting for `2.0.3` 🤞

**Comment by kristjanpirnpuu** [comment]
Fixed in 2.0.3
Available from cocoapods

**Comment by domasn** [comment]
Perfect. Can confirm it's fixed 👍
